src/js/components/AppVersionComponent.jsx

```diff
@@ -10,36 +10,58 @@ var UNSPECIFIED_NODE =
     }
   });
 /*eslint-enable react/display-name */
+
+function invalidateValue(value, suffix) {
+  if (value == null) {
+    return (
+      <UNSPECIFIED_NODE />
+    );
+  } else {
+    return (
+      <dd>{value} {suffix}</dd>
+    );
+  }
+}
```

Comment:
This shouldn't block merge btw...but it seems like we could just remove that rule as I think it's valuable to return jsx as I have on my above example.